src/main/java/fr/inria/jtravis/helpers/EntityHelper.java

```diff
@@ -78,9 +74,43 @@ public EntityHelper(JTravis jTravis) {
             }
         } catch (IOException e) {
             this.getLogger().error("Error while getting JSON at URL: "+url, e);
-        }
 
-        return Optional.empty();
+            // In case of 404 when getting repository or build, the repo might had been renamed
+            if ((zeClass == Repository.class || zeClass == Builds.class) && e.getMessage().contains("404 Not Found")) {
```

Comment:
I don't agree with this change. Do you agree that I push on your own branch?

Reply:
Please, be my guess.